src/factorization.jl

```diff
@@ -739,6 +739,7 @@ patterns with “more structure”.
 Base.@kwdef struct UMFPACKFactorization <: AbstractFactorization
     reuse_symbolic::Bool = true
     check_pattern::Bool = true # Check factorization re-use
+    control::Vector{Float64}=SparseArrays.UMFPACK.get_umfpack_control(Float64,Int64) # Control Parameters of UMFPACK 
```

Comment:
This will heap allocate each time in the default algorithm. Instead, make a const of the default, set the default here to `nothing`, and then if `nothing` use the const global.